bug: subscription to many static sharding topics hangs

**Describe the bug**
During TestStaticShardingLimits execution 1024 pubsub topics are subscribed. First batch of topics are named topics and there are no problems. When static sharding topics are subscribed, only few will get through. Entire subscription and ENR update process will hang. 

**To Reproduce**
1) Checkout https://github.com/waku-org/go-waku/pull/1060/commits/873caa8aa09a487df878301975ac018372a3ad59
2) cd go-waku/waku/v2/node
3) go test -run TestStaticShardingLimits

**Expected behavior**
Better handling for concurrent events or explanation at which rate is allowed to subscribe to static sharding topics.

**Log**

https://github.com/waku-org/go-waku/actions/runs/8537658954/job/23388721937?pr=1060

```
2024-04-03T19:00:45.549+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/my-app/21/1022", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:45.549+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/my-app/21/1022"}
2024-04-03T19:00:45.550+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/my-app/21/1022"}
2024-04-03T19:00:45.550+0800	INFO	gowaku.node2.peer-manager	peermanager/topic_event_handler.go:25	handleNewRelayTopicSubscription	{"pubSubTopic": "/waku/2/my-app/21/1022"}
2024-04-03T19:00:45.560+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/my-app/21/1023", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:45.561+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/my-app/21/1023"}
2024-04-03T19:00:45.561+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/my-app/21/1023"}
2024-04-03T19:00:45.561+0800	INFO	gowaku.node2.peer-manager	peermanager/topic_event_handler.go:25	handleNewRelayTopicSubscription	{"pubSubTopic": "/waku/2/my-app/21/1023"}
2024-04-03T19:00:45.572+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/0", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:45.572+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/0"}
2024-04-03T19:00:45.572+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/0"}
2024-04-03T19:00:45.572+0800	INFO	gowaku.node2.peer-manager	peermanager/topic_event_handler.go:25	handleNewRelayTopicSubscription	{"pubSubTopic": "/waku/2/rs/21/0"}
2024-04-03T19:00:45.572+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:45.572+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[0]}}
2024-04-03T19:00:45.573+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:45.573+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[0]}}
2024-04-03T19:00:45.574+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-KG4QL8xa6LLgt-QROwAl0wOho0NOk-k74MNAV0FDKoQXZN5G3p_R9-rucBjVWmiE3-DrzszmtXqo_ZgLefuZS3O7QyGAY6jnhiYgmlkgnY0gmlwhMCoAW6CcnOFABUBAACJc2VjcDI1NmsxoQMB6xWOgml7xHeOSiCGinJf2LMhrAEMBXZhuko1xa_bmIN0Y3CC7F6DdWRwgufAhXdha3UyAQ"}
2024-04-03T19:00:45.576+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-KG4QNkSAuVNPCmMBolxW5Svoy6VrFova1_PgmzNjZR3m3fPJsy6yPl7ONaQcqMOurHlJPmo83tQsQxw1nrdAh7Y-FCGAY6jnhiZgmlkgnY0gmlwhMCoAW6CcnOFABUBAACJc2VjcDI1NmsxoQMB6xWOgml7xHeOSiCGinJf2LMhrAEMBXZhuko1xa_bmIN0Y3CC7F6DdWRwgufAhXdha3UyAQ"}
2024-04-03T19:00:45.673+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/1", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:45.673+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/1"}
2024-04-03T19:00:45.673+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/1"}
2024-04-03T19:00:45.674+0800	INFO	gowaku.node2.peer-manager	peermanager/topic_event_handler.go:25	handleNewRelayTopicSubscription	{"pubSubTopic": "/waku/2/rs/21/1"}
2024-04-03T19:00:45.674+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:45.674+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[0,1]}}
2024-04-03T19:00:45.674+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:45.674+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[1,0]}}
2024-04-03T19:00:45.675+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-KO4QBIncAv37KwGoB-qvZhNkbl69as-kXM0VMJIaixQa7eLNKMb8V4LxE18ICkfvuqU69d2DttkkUVIANGZKaAeHoCGAY6jnhiagmlkgnY0gmlwhMCoAW6CcnOHABUCAAAAAYlzZWNwMjU2azGhAwHrFY6CaXvEd45KIIaKcl_YsyGsAQwFdmG6SjXFr9uYg3RjcILsXoN1ZHCC58CFd2FrdTIB"}
2024-04-03T19:00:45.676+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-KO4QAZXdC5Ayfa5KAvEY0HVCTdbFL39HPoaUCJZDxY1dKoXGu1memGtdg3mBDQK_6iWkQBi4PlsRgYtvp8iHqK2S5qGAY6jnhibgmlkgnY0gmlwhMCoAW6CcnOHABUCAAEAAIlzZWNwMjU2azGhAwHrFY6CaXvEd45KIIaKcl_YsyGsAQwFdmG6SjXFr9uYg3RjcILsXoN1ZHCC58CFd2FrdTIB"}
2024-04-03T19:00:45.775+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/2", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:45.775+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/2"}
2024-04-03T19:00:45.775+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/2"}
2024-04-03T19:00:45.775+0800	INFO	gowaku.node2.peer-manager	peermanager/topic_event_handler.go:25	handleNewRelayTopicSubscription	{"pubSubTopic": "/waku/2/rs/21/2"}
2024-04-03T19:00:45.776+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:45.776+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[1,2,0]}}
2024-04-03T19:00:45.776+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:45.776+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[1,2,0]}}
2024-04-03T19:00:45.777+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-KW4QJkjZ2XO4LC562Hvm1L41dHj0onO0q2BUSjNuBebLltXaJsnI66L2cRusA0Dbs2TPl2oLELDZdChxMYrLKQrHPKGAY6jnhicgmlkgnY0gmlwhMCoAW6CcnOJABUDAAEAAgAAiXNlY3AyNTZrMaEDAesVjoJpe8R3jkoghopyX9izIawBDAV2YbpKNcWv25iDdGNwguxeg3VkcILnwIV3YWt1MgE"}
2024-04-03T19:00:45.779+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-KW4QGcTemlg6kW_PdQyWw1qJKFIDhISb-rVhC4b9JEo0UVFbRWcR6lpmhCDXyabiAMVx7WrSBHG9xlPfTpPexdIedWGAY6jnhidgmlkgnY0gmlwhMCoAW6CcnOJABUDAAEAAgAAiXNlY3AyNTZrMaEDAesVjoJpe8R3jkoghopyX9izIawBDAV2YbpKNcWv25iDdGNwguxeg3VkcILnwIV3YWt1MgE"}
2024-04-03T19:00:45.876+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/3", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:45.876+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/3"}
2024-04-03T19:00:45.877+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/3"}
2024-04-03T19:00:45.877+0800	INFO	gowaku.node2.peer-manager	peermanager/topic_event_handler.go:25	handleNewRelayTopicSubscription	{"pubSubTopic": "/waku/2/rs/21/3"}
2024-04-03T19:00:45.877+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:45.877+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[1,2,3,0]}}
2024-04-03T19:00:45.878+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Ke4QOnYpbMzIb18BW3womMGnrDGh2u2gUOA8uFghj48f2NUXLkSmCEH_Fh81KPaI_p3KOqPOgYy0kwgEwbYLcIJHVeGAY6jnhiegmlkgnY0gmlwhMCoAW6CcnOLABUEAAEAAgADAACJc2VjcDI1NmsxoQMB6xWOgml7xHeOSiCGinJf2LMhrAEMBXZhuko1xa_bmIN0Y3CC7F6DdWRwgufAhXdha3UyAQ"}
2024-04-03T19:00:45.878+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:45.878+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[2,3,0,1]}}
2024-04-03T19:00:45.880+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Ke4QEg5bReYcgo67k9ZGTsDmsyRfg_QCwFwGCiDJxSw_-QARB0AxX7MvJMAB_NsmNGSVd4ui77WRIWzHhhq_lTuozqGAY6jnhifgmlkgnY0gmlwhMCoAW6CcnOLABUEAAIAAwAAAAGJc2VjcDI1NmsxoQMB6xWOgml7xHeOSiCGinJf2LMhrAEMBXZhuko1xa_bmIN0Y3CC7F6DdWRwgufAhXdha3UyAQ"}
2024-04-03T19:00:45.977+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/4", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:45.977+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/4"}
2024-04-03T19:00:45.977+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/4"}
2024-04-03T19:00:45.977+0800	INFO	gowaku.node2.peer-manager	peermanager/topic_event_handler.go:25	handleNewRelayTopicSubscription	{"pubSubTopic": "/waku/2/rs/21/4"}
2024-04-03T19:00:45.978+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:45.978+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[1,2,3,0,4]}}
2024-04-03T19:00:45.978+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:45.979+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[4,1,2,3,0]}}
2024-04-03T19:00:45.979+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Km4QKWRXQ9i9um20dphJwQB_wqbktQzXJBX4ei3H9_LvaIkOsZ9Yx84VCKYuriICAzCtAe_B8urASObRdN__GBtmj-GAY6jnhiggmlkgnY0gmlwhMCoAW6CcnONABUFAAEAAgADAAAABIlzZWNwMjU2azGhAwHrFY6CaXvEd45KIIaKcl_YsyGsAQwFdmG6SjXFr9uYg3RjcILsXoN1ZHCC58CFd2FrdTIB"}
2024-04-03T19:00:45.980+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Km4QFlHRcgzw_dW0GNyeie0YRlh14HTdXhLkwWPfgHJnvwYRIZtCsWV-WQmrhTHECG8-KHsJvd25lBOW9aaOXM3JKaGAY6jnhihgmlkgnY0gmlwhMCoAW6CcnONABUFAAQAAQACAAMAAIlzZWNwMjU2azGhAwHrFY6CaXvEd45KIIaKcl_YsyGsAQwFdmG6SjXFr9uYg3RjcILsXoN1ZHCC58CFd2FrdTIB"}
2024-04-03T19:00:46.079+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/5", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:46.079+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/5"}
2024-04-03T19:00:46.079+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/5"}
2024-04-03T19:00:46.079+0800	INFO	gowaku.node2.peer-manager	peermanager/topic_event_handler.go:25	handleNewRelayTopicSubscription	{"pubSubTopic": "/waku/2/rs/21/5"}
2024-04-03T19:00:46.079+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.079+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[0,5,4,1,2,3]}}
2024-04-03T19:00:46.080+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Ku4QJSDWKc_hdFzkjn_JnsdgiL9iWKGSGCkV9HGnK6jhCb5cp2LZH9UzVPBZ_GIOgPsI6cGemlOd_q6-dkxGyFooT2GAY6jnhiigmlkgnY0gmlwhMCoAW6CcnOPABUGAAAABQAEAAEAAgADiXNlY3AyNTZrMaEDAesVjoJpe8R3jkoghopyX9izIawBDAV2YbpKNcWv25iDdGNwguxeg3VkcILnwIV3YWt1MgE"}
2024-04-03T19:00:46.081+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.081+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[5,4,1,2,3,0]}}
2024-04-03T19:00:46.082+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Ku4QCzYSBPkMYr1wG4oVdATrxff7oUosE8Xn-1kI10c3WoRQ6nqpjC9MtbnKMqFK4AWMvZVJnjoRyuFBTlliLiyKcyGAY6jnhijgmlkgnY0gmlwhMCoAW6CcnOPABUGAAUABAABAAIAAwAAiXNlY3AyNTZrMaEDAesVjoJpe8R3jkoghopyX9izIawBDAV2YbpKNcWv25iDdGNwguxeg3VkcILnwIV3YWt1MgE"}
2024-04-03T19:00:46.179+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/6", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:46.179+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/6"}
2024-04-03T19:00:46.179+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/6"}
2024-04-03T19:00:46.180+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.180+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[5,4,1,2,6,3,0]}}
2024-04-03T19:00:46.180+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.180+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[1,2,6,3,0,5,4]}}
2024-04-03T19:00:46.181+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-K24QI4MFx8Urk75s7AJ_ZoZ6Q6IqcQWni81gudLn35bpNgJVhp9juD6OZQcpXoyzqwGjF-p9_yJ0L0TitJQPCS6QiqGAY6jnhikgmlkgnY0gmlwhMCoAW6CcnORABUHAAEAAgAGAAMAAAAFAASJc2VjcDI1NmsxoQMB6xWOgml7xHeOSiCGinJf2LMhrAEMBXZhuko1xa_bmIN0Y3CC7F6DdWRwgufAhXdha3UyAQ"}
2024-04-03T19:00:46.280+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/7", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:46.281+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/7"}
2024-04-03T19:00:46.281+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/7"}
2024-04-03T19:00:46.281+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.281+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[6,7,3,0,5,4,1,2]}}
2024-04-03T19:00:46.282+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-K-4QIhMJmYgrqM9-CE107d4oIzkesAIp42uce4LSUUPoZg0AY6cxw7qpHym9Pjcr0ReKNmc83T7UdD1vAdx6pBfpImGAY6jnhilgmlkgnY0gmlwhMCoAW6CcnOTABUIAAYABwADAAAABQAEAAEAAolzZWNwMjU2azGhAwHrFY6CaXvEd45KIIaKcl_YsyGsAQwFdmG6SjXFr9uYg3RjcILsXoN1ZHCC58CFd2FrdTIB"}
2024-04-03T19:00:46.284+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.284+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[6,7,3,0,5,4,1,2]}}
2024-04-03T19:00:46.285+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-K-4QI_015aYFyVLjo7718fNJFSFXDbt0kKR-5qTko5uedjCLmruVLbPPwBRGrjwQF-NoGulllXXgvkhp2c8e-S-X2KGAY6jnhimgmlkgnY0gmlwhMCoAW6CcnOTABUIAAYABwADAAAABQAEAAEAAolzZWNwMjU2azGhAwHrFY6CaXvEd45KIIaKcl_YsyGsAQwFdmG6SjXFr9uYg3RjcILsXoN1ZHCC58CFd2FrdTIB"}
2024-04-03T19:00:46.382+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/8", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:46.382+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/8"}
2024-04-03T19:00:46.382+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/8"}
2024-04-03T19:00:46.383+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.383+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[7,1,5,8,6,4,2,3,0]}}
2024-04-03T19:00:46.383+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.383+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[0,6,1,3,5,7,2,8,4]}}
2024-04-03T19:00:46.384+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-LG4QIiMuls49q-fvdpn4XwhgY3fPw6mv0ztPZm3esxqiB3TFsvygv8D_mDxLqhSn9W59pOdIcTX94CoNUkpup93h3SGAY6jnhingmlkgnY0gmlwhMCoAW6CcnOVABUJAAcAAQAFAAgABgAEAAIAAwAAiXNlY3AyNTZrMaEDAesVjoJpe8R3jkoghopyX9izIawBDAV2YbpKNcWv25iDdGNwguxeg3VkcILnwIV3YWt1MgE"}
2024-04-03T19:00:46.385+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-LG4QPzIIXD8EGOGKOKeaVdh6t3bEBiVNBMtitTtheKhSqXIPYt6tbY7hefxNYbp4ILQfp7xJe9WDZ1vHiCK62wLNY2GAY6jnhiogmlkgnY0gmlwhMCoAW6CcnOVABUJAAAABgABAAMABQAHAAIACAAEiXNlY3AyNTZrMaEDAesVjoJpe8R3jkoghopyX9izIawBDAV2YbpKNcWv25iDdGNwguxeg3VkcILnwIV3YWt1MgE"}
2024-04-03T19:00:46.483+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/9", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:46.483+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/9"}
2024-04-03T19:00:46.483+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/9"}
2024-04-03T19:00:46.483+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.483+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[7,0,9,1,3,8,4,6,2,5]}}
2024-04-03T19:00:46.484+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.484+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[8,1,2,7,0,6,5,4,9,3]}}
2024-04-03T19:00:46.484+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-LO4QC5JFqqlW10QWj4GMCAwJW8CO5ZtUDkTFTn5wMsYz1HJbRtKug1oDZ97NnGRvpuXKCmM59lDxAd7l7pbvnbrXraGAY6jnhipgmlkgnY0gmlwhMCoAW6CcnOXABUKAAcAAAAJAAEAAwAIAAQABgACAAWJc2VjcDI1NmsxoQMB6xWOgml7xHeOSiCGinJf2LMhrAEMBXZhuko1xa_bmIN0Y3CC7F6DdWRwgufAhXdha3UyAQ"}
2024-04-03T19:00:46.488+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-LO4QHrBYrdhAu9bA3mk2VbnNHPilsxtUseJfvzuUh8v-m44Az4iwilOjnuq5gK_Mk1f5cTEfiw3nAOsf-YiCKAEMXeGAY6jnhiqgmlkgnY0gmlwhMCoAW6CcnOXABUKAAgAAQACAAcAAAAGAAUABAAJAAOJc2VjcDI1NmsxoQMB6xWOgml7xHeOSiCGinJf2LMhrAEMBXZhuko1xa_bmIN0Y3CC7F6DdWRwgufAhXdha3UyAQ"}
2024-04-03T19:00:46.584+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/10", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:46.584+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/10"}
2024-04-03T19:00:46.585+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/10"}
2024-04-03T19:00:46.585+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.585+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[8,6,2,10,5,9,4,7,1,3,0]}}
2024-04-03T19:00:46.586+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.586+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[5,2,7,10,0,1,9,8,4,3,6]}}
2024-04-03T19:00:46.586+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-LW4QCWKCBixZ702M-3hf_XaoomodzF3e6J2xMuwfzuw9PQoHO9qpNmOpssnLUoLqwaNbHbL3YwLpev5EEY0oPVjFiaGAY6jnhirgmlkgnY0gmlwhMCoAW6CcnOZABULAAgABgACAAoABQAJAAQABwABAAMAAIlzZWNwMjU2azGhAwHrFY6CaXvEd45KIIaKcl_YsyGsAQwFdmG6SjXFr9uYg3RjcILsXoN1ZHCC58CFd2FrdTIB"}
2024-04-03T19:00:46.588+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-LW4QHXRpepX7XWDLnB9x3PE2as-hGz3A5Q21XNuDE7MWC2BJegCJ72pYARuIOVF7H_nV5eMtQsUGuH7QT_VsLh8xpqGAY6jnhisgmlkgnY0gmlwhMCoAW6CcnOZABULAAUAAgAHAAoAAAABAAkACAAEAAMABolzZWNwMjU2azGhAwHrFY6CaXvEd45KIIaKcl_YsyGsAQwFdmG6SjXFr9uYg3RjcILsXoN1ZHCC58CFd2FrdTIB"}
2024-04-03T19:00:46.685+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/11", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:46.685+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/11"}
2024-04-03T19:00:46.685+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/11"}
2024-04-03T19:00:46.686+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.686+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[1,11,6,9,7,10,5,4,2,3,0,8]}}
2024-04-03T19:00:46.686+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Le4QGoMLH5PN4pFrhzFMBpJm_PL6Dy5g9y07uxxZa4WGKVTL4xUOhYjO7rMO0cq23e16zqsdougwd-luEjeFlu3s0iGAY6jnhitgmlkgnY0gmlwhMCoAW6CcnObABUMAAEACwAGAAkABwAKAAUABAACAAMAAAAIiXNlY3AyNTZrMaEDAesVjoJpe8R3jkoghopyX9izIawBDAV2YbpKNcWv25iDdGNwguxeg3VkcILnwIV3YWt1MgE"}
2024-04-03T19:00:46.687+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.688+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[3,9,5,8,4,7,6,2,0,1,11,10]}}
2024-04-03T19:00:46.689+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Le4QDcKMBXWoGhbk2IHXuiueKAQsUZGZJ7S7C0n4Z0Fh1JjYyTKJqgz8DBh8d5aIK_JkVJAd5bAftf38cl5QXvBKryGAY6jnhiugmlkgnY0gmlwhMCoAW6CcnObABUMAAMACQAFAAgABAAHAAYAAgAAAAEACwAKiXNlY3AyNTZrMaEDAesVjoJpe8R3jkoghopyX9izIawBDAV2YbpKNcWv25iDdGNwguxeg3VkcILnwIV3YWt1MgE"}
2024-04-03T19:00:46.787+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/12", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:46.787+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/12"}
2024-04-03T19:00:46.787+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/12"}
2024-04-03T19:00:46.787+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.787+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[7,4,12,10,5,6,3,0,1,2,11,8,9]}}
2024-04-03T19:00:46.788+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.788+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[5,8,3,12,7,4,6,0,10,2,9,1,11]}}
2024-04-03T19:00:46.788+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Lm4QKdsuOi0SKMMTS9NycL2ZAHT9wdWw00Uu-EWEifKi3DEKIc7zN77e4Q-44wbWMiazNTCyhbwxipntveIBlHtUU-GAY6jnhivgmlkgnY0gmlwhMCoAW6CcnOdABUNAAcABAAMAAoABQAGAAMAAAABAAIACwAIAAmJc2VjcDI1NmsxoQMB6xWOgml7xHeOSiCGinJf2LMhrAEMBXZhuko1xa_bmIN0Y3CC7F6DdWRwgufAhXdha3UyAQ"}
2024-04-03T19:00:46.790+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Lm4QPyKkTzicBf8EQBYcGGilA1qAJSa7gnMH3ZMOf9jS3HAJHQufVsgc97YQOFW3yenb8glRMIekkwAzDwmAIb9fmKGAY6jnhiwgmlkgnY0gmlwhMCoAW6CcnOdABUNAAUACAADAAwABwAEAAYAAAAKAAIACQABAAuJc2VjcDI1NmsxoQMB6xWOgml7xHeOSiCGinJf2LMhrAEMBXZhuko1xa_bmIN0Y3CC7F6DdWRwgufAhXdha3UyAQ"}
2024-04-03T19:00:46.887+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/13", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:46.887+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/13"}
2024-04-03T19:00:46.887+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/13"}
2024-04-03T19:00:46.888+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.888+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[4,11,12,9,2,5,7,8,10,3,6,1,13,0]}}
2024-04-03T19:00:46.888+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.888+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[9,11,5,8,1,7,13,0,2,12,6,10,3,4]}}
2024-04-03T19:00:46.889+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Lu4QIoGe19rN6G-Yu2olHI8KAtR8_f8YOunJ0VSzUOOOzMudOzhNwhytH1i49KvR2pbl2ouR43QEhiFRWiT6vAkxOGGAY6jnhixgmlkgnY0gmlwhMCoAW6CcnOfABUOAAQACwAMAAkAAgAFAAcACAAKAAMABgABAA0AAIlzZWNwMjU2azGhAwHrFY6CaXvEd45KIIaKcl_YsyGsAQwFdmG6SjXFr9uYg3RjcILsXoN1ZHCC58CFd2FrdTIB"}
2024-04-03T19:00:46.891+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Lu4QI-VgdPpNdFkrEP69G8MVtNknKbcNd6LCTO1_lI5fY-LJ1qmN7eF74i5OKpti_7SwMdGqi4ixwTq7nedSWZ8MGOGAY6jnhiygmlkgnY0gmlwhMCoAW6CcnOfABUOAAkACwAFAAgAAQAHAA0AAAACAAwABgAKAAMABIlzZWNwMjU2azGhAwHrFY6CaXvEd45KIIaKcl_YsyGsAQwFdmG6SjXFr9uYg3RjcILsXoN1ZHCC58CFd2FrdTIB"}
2024-04-03T19:00:46.989+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/14", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:46.989+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/14"}
2024-04-03T19:00:46.989+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/14"}
2024-04-03T19:00:46.989+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.989+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[6,0,4,3,7,2,1,9,5,8,12,11,14,10,13]}}
2024-04-03T19:00:46.989+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:46.989+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[8,9,13,2,10,5,3,1,0,12,11,7,14,4,6]}}
2024-04-03T19:00:46.990+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-L24QEVyHgSRfQpYGPRoLPkeKLkLsyPpmh1qfGrwOquKKKOHK3n5vi84elE5qJkFsT6Ce1Op4y1vEnL_5ztyZI6Ws0aGAY6jnhizgmlkgnY0gmlwhMCoAW6CcnOhABUPAAYAAAAEAAMABwACAAEACQAFAAgADAALAA4ACgANiXNlY3AyNTZrMaEDAesVjoJpe8R3jkoghopyX9izIawBDAV2YbpKNcWv25iDdGNwguxeg3VkcILnwIV3YWt1MgE"}
2024-04-03T19:00:46.992+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-L24QCL-SQv4kWCQjdPNc_MLcCLaQJ6kfM0TTYJcpAzZUKfNNvHiOgXpOAd0ntpNsZAcIfk_rd-rS2vRknxu7k9L6UCGAY6jnhi0gmlkgnY0gmlwhMCoAW6CcnOhABUPAAgACQANAAIACgAFAAMAAQAAAAwACwAHAA4ABAAGiXNlY3AyNTZrMaEDAesVjoJpe8R3jkoghopyX9izIawBDAV2YbpKNcWv25iDdGNwguxeg3VkcILnwIV3YWt1MgE"}
2024-04-03T19:00:47.090+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/15", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:47.090+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/15"}
2024-04-03T19:00:47.090+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/15"}
2024-04-03T19:00:47.090+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:47.090+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[8,9,2,7,3,0,10,5,1,13,6,15,11,4,14,12]}}
2024-04-03T19:00:47.092+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-L-4QNDOedrUqOqEmay_41xJWgj4vjFToEeM4uwGlCl1iE1xL1v5j3foMzuZk2Owq4sm65ZZIkRJfrMwGmWJyEn_CKiGAY6jnhi1gmlkgnY0gmlwhMCoAW6CcnOjABUQAAgACQACAAcAAwAAAAoABQABAA0ABgAPAAsABAAOAAyJc2VjcDI1NmsxoQMB6xWOgml7xHeOSiCGinJf2LMhrAEMBXZhuko1xa_bmIN0Y3CC7F6DdWRwgufAhXdha3UyAQ"}
2024-04-03T19:00:47.092+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:47.092+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[13,10,8,12,14,4,1,9,0,7,15,2,6,3,5,11]}}
2024-04-03T19:00:47.094+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-L-4QLtFzlasLLELz-8KYkgzpqRVBo2XTd1tDFx2A3vvLsbgAcnlC7I56BTN4lbJj-FknxBSbA38G3m8krds4KAZwyeGAY6jnhi2gmlkgnY0gmlwhMCoAW6CcnOjABUQAA0ACgAIAAwADgAEAAEACQAAAAcADwACAAYAAwAFAAuJc2VjcDI1NmsxoQMB6xWOgml7xHeOSiCGinJf2LMhrAEMBXZhuko1xa_bmIN0Y3CC7F6DdWRwgufAhXdha3UyAQ"}
2024-04-03T19:00:47.190+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/16", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:47.190+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/16"}
2024-04-03T19:00:47.191+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/16"}
2024-04-03T19:00:47.191+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:47.191+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[7,0,10,6,8,9,3,4,5,1,16,14,15,12,2,13,11]}}
2024-04-03T19:00:47.191+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:47.191+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[9,0,2,1,3,15,13,4,14,10,8,16,7,6,5,11,12]}}
2024-04-03T19:00:47.192+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-MG4QN9P5Uljs9yE8fyhuQzIkugT1YMx7i9la8lUveTtkrEnZjv1qb6D-XcvLKlkFX13sHC2fdlg0K4-mwd_wr1OPgOGAY6jnhi3gmlkgnY0gmlwhMCoAW6CcnOlABURAAcAAAAKAAYACAAJAAMABAAFAAEAEAAOAA8ADAACAA0AC4lzZWNwMjU2azGhAwHrFY6CaXvEd45KIIaKcl_YsyGsAQwFdmG6SjXFr9uYg3RjcILsXoN1ZHCC58CFd2FrdTIB"}
2024-04-03T19:00:47.195+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-MG4QKEUL0DWHcdBemF-hox8xULZn9ytd9FaVtGf7PvsvV8UIYBnsahyCyFvEgB8ifXK_0f_LgQF79bsWKo6mV6USXGGAY6jnhi4gmlkgnY0gmlwhMCoAW6CcnOlABURAAkAAAACAAEAAwAPAA0ABAAOAAoACAAQAAcABgAFAAsADIlzZWNwMjU2azGhAwHrFY6CaXvEd45KIIaKcl_YsyGsAQwFdmG6SjXFr9uYg3RjcILsXoN1ZHCC58CFd2FrdTIB"}
2024-04-03T19:00:47.291+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/17", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:47.291+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/17"}
2024-04-03T19:00:47.292+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/17"}
2024-04-03T19:00:47.292+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:47.292+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[5,14,4,7,11,10,9,1,3,15,8,16,12,2,0,17,13,6]}}
2024-04-03T19:00:47.292+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:47.293+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[13,15,8,16,2,1,7,0,17,12,10,5,3,6,9,4,11,14]}}
2024-04-03T19:00:47.293+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-MO4QIAX6rb39uLYBzyId1GrL5gofd-QUFKiEVun6Ds4xTeee9dpulycHENR0NIEwIolgZyWupkyrAxT3Kbxp6NxIK-GAY6jnhi5gmlkgnY0gmlwhMCoAW6CcnOnABUSAAUADgAEAAcACwAKAAkAAQADAA8ACAAQAAwAAgAAABEADQAGiXNlY3AyNTZrMaEDAesVjoJpe8R3jkoghopyX9izIawBDAV2YbpKNcWv25iDdGNwguxeg3VkcILnwIV3YWt1MgE"}
2024-04-03T19:00:47.295+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-MO4QNGnhwIv-Lm6PnWWeGIUGRhk_3sJJBfcRBI5zlJS2kqtXonUB0SQwzzHQbeQmR6CkXP3nsE75Cy9swruT3GI30qGAY6jnhi6gmlkgnY0gmlwhMCoAW6CcnOnABUSAA0ADwAIABAAAgABAAcAAAARAAwACgAFAAMABgAJAAQACwAOiXNlY3AyNTZrMaEDAesVjoJpe8R3jkoghopyX9izIawBDAV2YbpKNcWv25iDdGNwguxeg3VkcILnwIV3YWt1MgE"}
2024-04-03T19:00:47.393+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/18", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:47.393+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/18"}
2024-04-03T19:00:47.393+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/18"}
2024-04-03T19:00:47.394+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:47.394+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[3,6,7,2,5,1,0,14,12,18,10,9,15,16,11,17,13,8,4]}}
2024-04-03T19:00:47.394+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:47.394+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[9,8,7,12,14,18,6,17,13,11,10,3,0,5,16,4,1,15,2]}}
2024-04-03T19:00:47.395+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-MW4QC3TzkGXVOTIw8YWm3ewQqHjgfLJrBUoPFtyH3VXkAS6Mxb6aWGoMLuB6ahelExqVU6m6L7mKs8WrYE_SlJKLkuGAY6jnhi7gmlkgnY0gmlwhMCoAW6CcnOpABUTAAMABgAHAAIABQABAAAADgAMABIACgAJAA8AEAALABEADQAIAASJc2VjcDI1NmsxoQMB6xWOgml7xHeOSiCGinJf2LMhrAEMBXZhuko1xa_bmIN0Y3CC7F6DdWRwgufAhXdha3UyAQ"}
2024-04-03T19:00:47.397+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-MW4QA-W2GArzAIt9SjMLElT0jZXgOKUWpurhFRC01YR1cGQFCFCm2Tyruq6edG8QDnsQTJb5f5JpGeudlvmcMNB9eKGAY6jnhi8gmlkgnY0gmlwhMCoAW6CcnOpABUTAAkACAAHAAwADgASAAYAEQANAAsACgADAAAABQAQAAQAAQAPAAKJc2VjcDI1NmsxoQMB6xWOgml7xHeOSiCGinJf2LMhrAEMBXZhuko1xa_bmIN0Y3CC7F6DdWRwgufAhXdha3UyAQ"}
2024-04-03T19:00:47.495+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/19", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:47.495+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/19"}
2024-04-03T19:00:47.495+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/19"}
2024-04-03T19:00:47.495+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:47.495+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[1,12,17,4,5,8,15,2,0,10,11,7,3,6,18,16,14,19,9,13]}}
2024-04-03T19:00:47.496+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:47.496+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[5,17,4,13,0,15,1,16,19,6,8,11,2,10,3,18,12,7,14,9]}}
2024-04-03T19:00:47.496+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Me4QMjV54E2ay5JzMu89FMaJAhsgkkFA0poWCgbFFCH8sCoKjklFgH80aMvNI-MrHYWxgN2ZiMjnekky-cEgYOXmE6GAY6jnhi9gmlkgnY0gmlwhMCoAW6CcnOrABUUAAEADAARAAQABQAIAA8AAgAAAAoACwAHAAMABgASABAADgATAAkADYlzZWNwMjU2azGhAwHrFY6CaXvEd45KIIaKcl_YsyGsAQwFdmG6SjXFr9uYg3RjcILsXoN1ZHCC58CFd2FrdTIB"}
2024-04-03T19:00:47.498+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Me4QHJjtSF75hGPa9m6Rs2DiTKo9jy4kO7P3gnzRx4HWrZWDaUCNmT3vEs7D5OVKfG0MW1zZ3a5t2h-q_-_2o6YIW6GAY6jnhi-gmlkgnY0gmlwhMCoAW6CcnOrABUUAAUAEQAEAA0AAAAPAAEAEAATAAYACAALAAIACgADABIADAAHAA4ACYlzZWNwMjU2azGhAwHrFY6CaXvEd45KIIaKcl_YsyGsAQwFdmG6SjXFr9uYg3RjcILsXoN1ZHCC58CFd2FrdTIB"}
2024-04-03T19:00:47.596+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/20", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:47.596+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/20"}
2024-04-03T19:00:47.596+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/20"}
2024-04-03T19:00:47.597+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:47.597+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[18,1,12,16,5,2,6,9,17,4,10,20,14,8,13,3,15,7,19,11,0]}}
2024-04-03T19:00:47.598+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Mm4QHSNghE-IMDykclgZ_CphSW6E6c7vU4VQZIxF8qh9hWOBrZevrYcLGZQ2q7av8K3fhjs8rO6ZCDO_y84aCze_zmGAY6jnhi_gmlkgnY0gmlwhMCoAW6CcnOtABUVABIAAQAMABAABQACAAYACQARAAQACgAUAA4ACAANAAMADwAHABMACwAAiXNlY3AyNTZrMaEDAesVjoJpe8R3jkoghopyX9izIawBDAV2YbpKNcWv25iDdGNwguxeg3VkcILnwIV3YWt1MgE"}
2024-04-03T19:00:47.598+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:47.598+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[15,13,20,8,0,5,2,11,6,17,19,10,9,16,3,1,14,7,12,18,4]}}
2024-04-03T19:00:47.599+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Mm4QD4IF78Piop_NeU73hDzRxUS89Cn3gxgqxlnU8tToetiebLBLDkpP4sPDaYftoYULzmsr7QSaR0inGxUj90f4buGAY6jnhjAgmlkgnY0gmlwhMCoAW6CcnOtABUVAA8ADQAUAAgAAAAFAAIACwAGABEAEwAKAAkAEAADAAEADgAHAAwAEgAEiXNlY3AyNTZrMaEDAesVjoJpe8R3jkoghopyX9izIawBDAV2YbpKNcWv25iDdGNwguxeg3VkcILnwIV3YWt1MgE"}
2024-04-03T19:00:47.697+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/21", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:47.697+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/21"}
2024-04-03T19:00:47.697+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/21"}
2024-04-03T19:00:47.698+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:47.698+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[2,19,8,4,6,13,12,17,21,18,1,9,11,7,0,15,14,5,10,16,3,20]}}
2024-04-03T19:00:47.698+0800	INFO	gowaku.node2	node/localnode.go:321	updating advertised relay shards in ENR
2024-04-03T19:00:47.698+0800	WARN	gowaku.node2	node/localnode.go:323	A mix of named and static shards found. ENR shard will contain only the following shards	{"shards": {"clusterID":21,"shardIDs":[5,14,9,21,16,13,4,10,20,19,1,0,6,17,8,7,18,12,15,2,11,3]}}
2024-04-03T19:00:47.698+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Mu4QNJ-yTzAuNrLyR7muAKrogJyWusWegOmpkGG10PkbvTfFLDB3ze8EDGN4ycetENy-uJ6o--cGsP9xsmnsg6NyvWGAY6jnhjBgmlkgnY0gmlwhMCoAW6CcnOvABUWAAIAEwAIAAQABgANAAwAEQAVABIAAQAJAAsABwAAAA8ADgAFAAoAEAADABSJc2VjcDI1NmsxoQMB6xWOgml7xHeOSiCGinJf2LMhrAEMBXZhuko1xa_bmIN0Y3CC7F6DdWRwgufAhXdha3UyAQ"}
2024-04-03T19:00:47.700+0800	INFO	gowaku.node2	node/wakunode2.go:553	new enr record	{"enr": "enr:-Mu4QGAx8ML1d9NpDp49VDO1vL262Yzsn0RkauDTGyZxqiqTCq9lhXt_dngTuZEgOKZQtUwF24P9K94JFElxjFIwTJeGAY6jnhjCgmlkgnY0gmlwhMCoAW6CcnOvABUWAAUADgAJABUAEAANAAQACgAUABMAAQAAAAYAEQAIAAcAEgAMAA8AAgALAAOJc2VjcDI1NmsxoQMB6xWOgml7xHeOSiCGinJf2LMhrAEMBXZhuko1xa_bmIN0Y3CC7F6DdWRwgufAhXdha3UyAQ"}
2024-04-03T19:00:47.799+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/22", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:00:47.799+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/22"}
2024-04-03T19:02:50.986+0800	ERROR	gowaku.node2.relay	relay/waku_relay.go:540	getting message from subscription	{"error": "context deadline exceeded"}

2024-04-03T19:02:50.993+0800	ERROR	gowaku.node2.relay	relay/waku_relay.go:540	getting message from subscription	{"error": "context deadline exceeded"}
2024-04-03T19:02:50.993+0800	ERROR	gowaku.node2.relay	relay/waku_relay.go:540	getting message from subscription	{"error": "context deadline exceeded"}
2024-04-03T19:02:51.118+0800	WARN	gowaku.node2.discv5	discv5/discover.go:486	Discv5 loop stopped
2024-04-03T19:02:51.567+0800	WARN	gowaku.node2.discv5	discv5/discover.go:486	Discv5 loop stopped
2024-04-03T19:03:07.142+0800	INFO	gowaku.node2.peer-manager	peermanager/topic_event_handler.go:25	handleNewRelayTopicSubscription	{"pubSubTopic": "/waku/2/rs/21/6"}
2024-04-03T19:03:07.142+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:247	gossipsub subscription	{"pubsubTopic": "/waku/2/rs/21/22"}
2024-04-03T19:03:07.244+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:406	subscribing to	{"pubsubTopic": "/waku/2/rs/21/23", "contentTopics": ["/test/2/my-app/sharded"]}
2024-04-03T19:03:07.244+0800	INFO	gowaku.node2.relay	relay/waku_relay.go:215	subscribing to underlying pubsubTopic	{"pubsubTopic": "/waku/2/rs/21/23"}
2024-04-03T19:03:07.244+0800	ERROR	gowaku.node2.relay	relay/waku_relay.go:186	failed to register topic validator	{"pubsubTopic": "/waku/2/rs/21/23", "error": "context deadline exceeded"}
2024-04-03T19:03:07.244+0800	ERROR	gowaku.node2.relay	relay/waku_relay.go:221	failed to upsert topic	{"pubsubTopic": "/waku/2/rs/21/23", "error": "context deadline exceeded"}
2024-04-03T19:03:07.244+0800	ERROR	gowaku.node2.relay	relay/waku_relay.go:416	failed to subscribe to pubsubTopic	{"error": "context deadline exceeded", "pubsubTopic": "/waku/2/rs/21/23"}
2024-04-03T19:03:07.244+0800	INFO	gowaku.node2.discv5	discv5/discover.go:248	stopped Discovery V5
2024-04-03T19:03:07.246+0800	INFO	gowaku.node2.discv5	discv5/discover.go:248	stopped Discovery V5
--- FAIL: TestStaticShardingLimits (156.26s)
    wakunode2_test.go:482:
        	Error Trace:	/Users/roman/sources/waku-org/go-waku/waku/v2/node/wakunode2_test.go:482
        	Error:      	Received unexpected error:
        	            	context deadline exceeded
        	Test:       	TestStaticShardingLimits
FAIL
exit status 1
FAIL	github.com/waku-org/go-waku/waku/v2/node	157.529s
```

I've added shardIDs check based on actual ENR value to help distinguish between false error and real failure to update ENR. 
Please checkout https://github.com/waku-org/go-waku/pull/1060/commits/b6fc2cb6840fb08940e1009cdc4b874566273bff

https://github.com/waku-org/go-waku/actions/runs/8593720446/job/23545650778?pr=1060
```
2024-04-08T02:33:07.194Z	WARN	gowaku.node2.discv5	discv5/discover.go:486	Discv5 loop stopped
2024-04-08T02:33:07.194Z	INFO	gowaku.node2.discv5	discv5/discover.go:248	stopped Discovery V5
--- FAIL: TestStaticShardingLimits (13.86s)
    wakunode2_test.go:486: 
        	Error Trace:	/home/runner/work/go-waku/go-waku/waku/v2/node/wakunode2_test.go:486
        	Error:      	Not equal: 
        	            	expected: 1024
        	            	actual  : 255
        	Test:       	TestStaticShardingLimits
FAIL

```

--- a/waku/v2/peermanager/peer_manager.go
+++ b/waku/v2/peermanager/peer_manager.go
@@ -304,7 +304,7 @@ func (pm *PeerManager) ensureMinRelayConnsPerTopic() {
 			notConnectedPeers := pm.getNotConnectedPers(topicStr)
 			if notConnectedPeers.Len() == 0 {
 				pm.logger.Debug("could not find any peers in peerstore to connect to, discovering more", zap.String("pubSubTopic", topicStr))
-				pm.discoverPeersByPubsubTopics([]string{topicStr}, relay.WakuRelayID_v200, pm.ctx, 2)
+				go pm.discoverPeersByPubsubTopics([]string{topicStr}, relay.WakuRelayID_v200, pm.ctx, 2)
 				continue
 			}
 			pm.logger.Debug("connecting to eligible peers in peerstore", zap.String("pubSubTopic", topicStr))

